### ISSUE TITLE
phpunit-core should not require Drupal installation

### ISSUE THREAD

**Issue by mradcliffe** [LionsAd/drupal_ti#38]
Running phpunit tests should not require a Drupal installation as there should be no database state for those tests. It should only need to download Drupal.

Solution should be to decouple download and install tasks from the ensure drupal and ensure module functions.

**Comment by LionsAd** [comment]
That is correct and I missed that the installation is done, too for phpunit-core. I usually used the phpunit runner with a dependency on drupal/drupal myself in cases where I only use tests in a dedicated tests directory, but I agree phpunit-core needlessly does that.

Thanks for the bug report!
